docs/physics/montecarlo/initialization.ipynb

```diff
@@ -71,10 +71,39 @@
   },
```

Comment:
<p>One last thing as a general coding strategy, <code>number</code> should be passed as an argument to <code>func1()</code> instead of having it set in the notebook -- it expands your ability to profile the function with various different settings, if you wanted to.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/2254/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>